task_recorder2_msgs/Description description
---
task_recorder2_msgs/DataSample data_sample
string info
int32 return_code
int32 SERVICE_CALL_FAILED = 0
int32 SERVICE_CALL_SUCCESSFUL = 1